sound/hybrid_font.h

```diff
@@ -120,7 +130,49 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  void StartSwing() override {
+    if (!guess_monophonic_) {
+      if (!swing_player_) {
+        swing_player_ = PlayPolyphonic(&swng);
+      } else {
+        second_swing_player_ = PlayPolyphonic(&swng);
+      }
+    } else {
+      PlayMonophonic(&swing, &hum);
+    }
+  }
+
+  float SetSwingVolume(float swing_strength, float mixhum) {
```

Comment:
In order for non-smoothswing fonts to use this, hybrid font has to call this function somewhere.
However, doing so will change font behaviors in ways which I'm not sure are always better.
If we think that updating the swing volume is almost always better than not doing it, that's fine, but otherwise we need the ProffieOSSwingVolumeSharpness to default to 0.0 and ProffiOSMaxSwingVolume to default to whatever the swing defaults to normally? (I don't remember if that is 1.0 or 0.5)

Reply:
Default swing volume is effectively 1.0.  I'll test some mono/poly fonts with volume changing in place and see how it works.

Reply:
Setting swing volume seems fine in the polyphonic NEC fonts I've tested.